ansible/OWNERS

```diff
@@ -0,0 +1,5 @@
+assignees:
+- rutsky
+- stephenrlouie
+- danehans
```

Comment:
@foxish is there any way to detect reviewers? Reviewer = user adding labels && merging PR && commenting PR